### PR TITLE
RHPAM-2976 Update table

### PR DESCRIPTION
**Pull request by Sgitario**
JIRA: https://issues.redhat.com/browse/RHPAM-2976
Description: Update the table of supported databases.

Please make sure your PR meets the following requirements:

- [x] Pull Request title is properly formatted: `[RHPAM-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

**Comment by Sgitario** [comment]
@spolti can you have a look to this PR? Thanks.

**Comment by Sgitario** [comment]
> seeems good to me, do you mind to align the table?

@spolti I see the table correctly formatted in the final view.

**Comment by spolti** [comment]
I know, I meant to align the table on the raw text, like:
```
| MySQL       | 5.7, 8.0                | mysql-connector-java-8.0.12.jar |
| PosgtgreSQL | 10.1, 10.1 Enterprise   | postgresql-42.2.5.jar           |	| PosgtgreSQL | 10.1, 11.5, 10.1 Enterprise, 11.6 Enterprise   | postgresql-42.2.5.jar           |
```

File: templates/contrib/jdbc/README.md [hunk 1]
```diff
@@ -45,15 +45,15 @@ The drivers images are based on the supported configurations for Red Hat Process
 
 | Database    | DB Version              | JDBC Driver jar/version         |
 |-------------|-------------------------|---------------------------------|
```
Comment (spolti) [inline templates/contrib/jdbc/README.md:47]: you forget the two frist lines

Comment (Sgitario) [reply templates/contrib/jdbc/README.md]: Ahrg! Sorry, fixed.

**Pull request merged**